resources/views/site/blocks/360_embed.blade.php

```diff
@@ -1,13 +1,13 @@
-@php
-    $image_sequence_file = $block->assetLibrary;
-    $sequence_id = $image_sequence_file['id'];
-    $alt_text = $block->input('alt_text');
-    $caption_title = $block->input('caption_title');
-    $caption = $block->input('caption');
-    $size = $block->input('size');
-@endphp
+@if (isset($image_sequence_file) && $image_sequence_file != null)
```

Comment:
`$image_sequence_file` isn't set until [line 4](https://github.com/art-institute-of-chicago/artic.edu/pull/456/files#diff-11fb881d69cdfc3eca656b4c4aa8e85253b332d52dd659690c0e451285f797ceR4) below, so this will always evaluate to `false`.